examples/input/mouse_position.rs

```diff
@@ -0,0 +1,43 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::build()
+        .add_default_plugins()
+        .add_startup_system(setup.system())
+        .add_system(mouse_position.system())
+        .run();
+}
+
+// Register ::Horizontal and ::Vertical resources, so they can be read
+fn setup(mut mouse_axis: ResMut<Axis<Mouse>>, mut cursor_axis: ResMut<Axis<Cursor>>) {
+    // Axis<Mouse> represents raw mouse movements, without acceleration
+    // but it's inconsistent across platforms
+    // same mouse move can have a different scale
+    // ideal for 3D camera movement etc.
+    mouse_axis.register(Mouse::Horizontal);
```

Comment:
I think I would prefer it if the "default" axes are registered automatically. This is a common enough case that we should handle it on behalf of users.